app/views/components/GoodsRecordFilterSearchTerm.scala.html

```diff
@@ -24,49 +24,45 @@
 
 @(form: Form[_], countries: Seq[Country], page: Int, searchForm: SearchForm)(implicit request: Request[_], messages: Messages)
 
-@if(searchForm.searchTerm.nonEmpty || searchForm.countryOfOrigin.nonEmpty || searchForm.statusValue.nonEmpty) {
+@if(searchForm.searchTerm.nonEmpty
+    || searchForm.countryOfOrigin.nonEmpty
+    || searchForm.statusValue.nonEmpty){
 
-<div class="govuk-!-margin-top-1">
-    <p class="govuk-body">@Html(messages("goodsRecords.relatedTo"))</p>
+    @Html(messages("goodsRecords.relatedTo"))
```

Comment:
what does this result in? I'm assuming since there's no tags around it, it might be incorrect html - unsure though